arkouda/series.py

```diff
@@ -75,28 +79,37 @@ class Series:
 
     Parameters
     ----------
-    ar_tuple : 2-tuple of arkouda arrays with the first being the grouping key(s) and the
-             second being the value. The grouping key(s) will be the index of the series.
+    ar_tuple : tuple, list.
+        2-tuple of arkouda arrays with the first being the grouping key(s) and the
+        second being the value. The grouping key(s) will be the index of the series.
+        This will be deprecated in future releases.
+    index : pdarray
+        an array of indexes associated with the data array.
+        If empty, it will default to a range of ints whose size match the size of the data.
+    data : pdarray, Strings, Categorical
+        a 1D array. Must not be empty if ar_tuple is not provided.
 
     """
 
-    def __init__(self, ar_tuple=None, data=None, index=None):
+    @typechecked
+    def __init__(self, ar_tuple: Union[tuple, list] = None, index: pdarray = None,
+                 data: groupable_element_type = None):
+        # TODO: Allow index to be an Index when index.py is updated
         if ar_tuple is not None:
             self.index = Index.factory(ar_tuple[0])
             self.values = ar_tuple[1]
+            warnings.warn('ar_tuple will be deprecated in future releases', DeprecationWarning)
```

Comment:
We might want to add to this warning message how to pass it with the new method so the user doesn't have to guess or look into the code